docs/development/core/development-functional-tests.asciidoc

```diff
@@ -133,6 +141,28 @@ The `FunctionalTestRunner`'s primary purpose is to execute test files. These fil
 **Test Suite**:::
 A test suite is a collection of tests defined by calling `describe()`, and then populated with tests and setup/teardown hooks by calling `it()`, `before()`, `beforeEach()`, etc. Every test file must define only one top level test suite, and test suites can have as many nested test suites as they like.
 
+**Cross-browser testing**:::
+All the functional tests are executed in Chrome by default. To run specific tests in both Chrome and Firefox, `smoke` tag should be assigned:
+
+["source","js"]
+-----------
+// tags must be defined in `describe` function
+describe('My Cross-browser Test Suite', function () {
+  this.tags('smoke');
```

Comment:
Just to clarify if a test does not have 'smoke' tag and someone runs the config for FF for that test, will it not run or will it run on Chrome?

Reply:
I need to rephrase it because currently, it is not clear. The only filtering `config.firefox.js` has is:
```
    suiteTags: {
      exclude: ['skipFirefox'],
    },
```

So if you run tests with this config only, it will run everything that has no 'skipFirefox', no matter if `smoke` set or not.